zzk/service/state.go

```diff
@@ -577,8 +635,24 @@ func DeleteState(conn client.Connection, req StateRequest) error {
 		logger.Debug("No state to delete on host")
 	}
 
-	// Delete the service instance
 	sspth := path.Join(basepth, "/services", req.ServiceID, req.StateID())
+
+	// Delete the current state (status)
```

Comment:
Yes, because if the node is not there, you don't want to add it to the transaction, otherwise it will error